src/pages/payment-methods/index.js

```diff
@@ -21,13 +21,21 @@ const Th = styled.th`
     padding: 1.6rem 2rem;
 
     :first-child {
+        position: sticky;
+        /* stylelint-disable-next-line value-no-vendor-prefix */
+        position: -webkit-sticky;
+        left: -5px; /* required */
+        background-color: white;
+        z-index: 2;
+
         p {
-            text-align: center;
+            text-align: center !important;
```

Comment:
can we avoid using important and style it in its specific component?

Reply:
Just double checked that !important is unnecessary